src/pybind/mgr/dashboard/frontend/src/app/shared/services/task-message.service.ts

```diff
@@ -511,26 +489,44 @@ export class TaskMessageService {
       (metadata) => this.snapshotSchedule(metadata)
     ),
     // smb
-    'smb/cluster/create': this.newTaskMessage(this.commonOperations.create, (metadata) =>
+    'cephfs/smb/cluster/create': this.newTaskMessage(this.commonOperations.create, (metadata) =>
```

Comment:
commented on anpther PR as well, your choice to update on anyone: need types for `metadata`

Reply:
added it